catalog/plugin/catalog-plugin-videothumbnail/src/main/java/org/codice/ddf/catalog/content/plugin/video/VideoThumbnailPlugin.java

```diff
@@ -76,7 +77,32 @@
 
     private static final int MAX_FFMPEG_PROCESSES = 4;
 
-    private static final PumpStreamHandler DEV_NULL = new PumpStreamHandler(null);
+    private static final PumpStreamHandler DEV_NULL = new PumpStreamHandler(null) {
+
+        @Override
+        public void setProcessErrorStream(InputStream inputStream) {
+            consumeStream(inputStream);
+        }
+
+        @Override
+        public void setProcessOutputStream(InputStream inputStream) {
+            consumeStream(inputStream);
+        }
+
+        private void consumeStream(InputStream inputStream) {
+            byte[] buf;
```

Comment:
Could we instantiate the array here so we don't have to create a new one for each iteration?